docs/guides/multiverse-computing-singularity.ipynb

```diff
@@ -86,38 +70,70 @@
    "source": [
     "## Action-based approach\n",
     "\n",
-    "The function uses an action-based approach. You can think of it as a virtual environment where you use actions to perform tasks or change its state. Currently, it offers the following actions: [list](#1-list), [create](#2-create), [delete](#3-delete), [fit](#4-fit), [predict](#5-predict), [fit_predict](#6-fit-predict), and [create_fit_predict](#7-create-fit-predict). The general structure is as follows:"
+    "The function uses an action-based approach. You can think of it as a virtual environment where you use actions to perform tasks or change its state. Currently, it offers the following actions: [list](#1-list), [create](#2-create), [delete](#3-delete), [fit](#4-fit), [predict](#5-predict), [fit_predict](#6-fit-predict), and [create_fit_predict](#7-create-fit-predict). See the example below that demonstrates the `create_fit_predict` action."
```

Comment:
```suggestion
    "The function uses an action-based approach. You can think of it as a virtual environment where you use actions to perform tasks or change its state. Currently, it offers the following actions: [list](#1-list), [create](#2-create), [delete](#3-delete), [fit](#4-fit), [predict](#5-predict), [fit_predict](#6-fit-predict), and [create_fit_predict](#7-create-fit-predict). The following example demonstrates the `create_fit_predict` action."
```
(changing to "following" rather than "below" for accessibility)